CrowdStrike/crowdstrike-telemetry/ingest/parser.yml

```
@@ -53,6 +65,20 @@ stages:
       - set:
           "dns.question.name": "{{parsed_event.message.DomainName}}"
           "dns.question.type": "{{parsed_event.message.RequestType}}"
+          "source.ip": "{{parsed_event.message.LocalAddressIP6}}"
+          "source.nat.port": "{{parsed_event.message.LocalPort}}"
+
+          "destination.ip": "{{parsed_event.message.RemoteAddressIP6}}"
+          "destination.nat.ip": "{{parsed_event.message.RemoteAddressIP4}}"
+          "destination.nat.port": "{{parsed_event.message.RemotePort | int}}"
+
+      - set:
+          "source.nat.ip": >
+            [
+              {% for record in parsed_event.message.get("aip").split(" ") %}
+                  {% if record != "" %}"{{record}}",{% endif %}
+              {% endfor %}
+            ]
```

[`source.nat.ip`](https://www.elastic.co/guide/en/ecs/current/ecs-source.html#field-source-nat-ip) doesn't hold an array. So, we will have a problem with this field.
However, [`host.ip`](https://www.elastic.co/guide/en/ecs/current/ecs-host.html#field-host-ip) accept an array and the meaning is now brolen.
So I suggest changing to `host.ip`:
```suggestion
          "host.ip": >
            [
              {% for record in parsed_event.message.get("aip").split(" ") %}
                  {% if record != "" %}"{{record}}",{% endif %}
              {% endfor %}
            ]
```